CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+## Configuring the Environment
+
+This guide was written for `python 3.8.0`. To set up a local environment for this project and have source (`src`) mirror production state, you will have to:
+
+1. Create a virtual environment in the root folder of the project with `python3 -m venv venv`. Activate the environment with `source venv/bin/activate` and `pip install dbt-core==1.5.2`.
+
+2. Ask the admin @tm41m for credentials to your personal postgres development database and have your ip whitelisted to access the VPC network.
+
+3. Copy the `env.sample` file into a new file called `env.local` and replace the variable's values with your personal credentials. Run `source .env.local`.
```

Comment:
Did you mean "Copy the '.env.sample' file into a new file called '.env.local.'"?